apps/src/templates/testPages/WebLabTest.jsx

```diff
@@ -0,0 +1,179 @@
+      [Status.SUCCEEDED]: 'Success',
+      [Status.FAILED]: 'Failed'
+    }[this.state[test]];
+
+    return <p>{`${statusText}`}</p>;
+  };
+
+  renderCallToAction = () => {
+    const testFailed = [
+      STATUS_CODE_PROJECTS,
+      STATUS_COMPUTING_IN_THE_CORE
+    ].some(test => this.state[test] === Status.FAILED);
+
+    if (testFailed) {
+      return (
+        <div>
+          <p>
+            <strong>Error. One or more checks failed.</strong>
+          </p>
+          <p>
+            <strong>Action required: </strong>
+            Please contact your IT department to update your school’s firewall
+            settings or device policies. Contact our Support Team (
+            <a href="mailto:support@code.org">support@code.org</a>) if you have
+            any questions.
+          </p>
+        </div>
+      );
+    }
+
+    return (
+      <p>
+        <strong>You passed all checks! No further action is needed.</strong>
+      </p>
+    );
+  };
+
+  render() {
+    return (
+      <div id="main-content" className="container">
+        <div className="row">
+          <div className="col-md-6">
+            <div className="form-group">
+              <h1>Web Lab Network Support Check</h1>
+              <p>
+                Use this page to test your school's network support for our Web
+                Lab modules. Web Lab communicates with domains which may be
+                blocked by some school systems’ networks or device policies. If
+                the network check is unsuccessful, you will need to reach out to
+                your school's IT department to update your school's firewall
+                settings or device policies.
+              </p>
+              <p>
+                When checking network support, try to meet as many of these
+                criteria as possible:
+              </p>
+              <li>Use your school's internet connection</li>
+              <li>Use a student computer</li>
+              <li>Log into the computer with a student account</li>
+              <br />
+              <p>
+                Note: You do not need to be logged in to Code.org for this test
+                to be successful.
+              </p>
+              <label htmlFor="connect">
+                Click "Connect" to check whether your school's network supports
+                the Web Lab tool
+              </label>
+              <div id="status-table">
+                <table>
+                  <tbody>
+                    <tr>
+                      <th>Check</th>
+                      <th>Status</th>
+                    </tr>
+                    <tr>
+                      <td>
+                        Connected to
+                        <code>https://downloads.computinginthecore.org</code>
+                      </td>
+                      <td className="computinginthecore">
+                        {this.renderTestStatus(STATUS_COMPUTING_IN_THE_CORE)}
+                      </td>
```

Comment:
nit: in the screenshots, this text seems a bit higher than the text on the cell on the left.  Visually, it seems they should have the same vertical location.

Reply:
Fixed!